packages/react-percy/src/args.js

```diff
@@ -1,10 +1,6 @@
 export const docs = 'Documentation: https://github.com/percy/react-percy';
 
 export const options = {
-  config: {
```

Comment:
Users will no longer pass in the path of their webpack config; all webpack settings are added via the `percy.config.js` (or auto-loaded in the case of `create-react-app`).